daemons/fenced/fenced_commands.c

```diff
@@ -407,11 +408,19 @@ stonith_device_execute(stonith_device_t * device)
         action_str = "off";
     }
 
+    if (is_set(device->flags, st_device_supports_parameter_port)) {
+        host_arg = "port";
+
+    } else if (is_set(device->flags, st_device_supports_parameter_plug)) {
+        host_arg = "plug";
+    }
```

Comment:
If both are specified, should we prefer port or plug?

Reply:
I'm kind of hesitant too... "port" for the best backward compatibility as how it is now, "plug" for the favor of the new standard though ...